pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -30,7 +30,12 @@ rec {
     else legacy_390;
 
   # No active beta right now
```

Comment:
Could you remove this comment, since it is not relevant now?

Reply:
Oh yeah, right, lol ^^

Done.